rzk-playground/components/editor/Editor.tsx

```diff
@@ -19,12 +19,20 @@ export default function Editor({
     editorHeight: number
 }) {
     const [existsSelection, setExistsSelection] = useState(false)
+    const params = new URLSearchParams(window.location.search);
```

Comment:
Since this is a Next.js (not normal React.js) project, `window` should not be used outside `useEffect` since it is not available during SSR. I can see that the whole project is wrapped in `next/dynamic` so it shouldn't be an issue in this specific situation, but that's not a good practice in general.
I wonder why you didn't just use the Vite template for React since we don't need any Next.js features.